src/common/checksums.h

```diff
@@ -48,15 +48,35 @@ class SyncJournalDb;
  */
 OCSYNC_EXPORT QByteArray findBestChecksum(const QByteArray &checksums);
 
+class OCSYNC_EXPORT ChecksumHeader
+{
+public:
+    /// Parses a checksum header
```

Comment:
This comment isn't all that helpful. It's been there before, though, so nevermind.